generator/generator_tool/generator_tool.cpp

```diff
@@ -50,6 +50,20 @@
 
 #include "3party/gflags/src/gflags/gflags.h"
 
+namespace
+{
+char const * GetDataPathHelp()
+{
+  static std::string const kHelp =
+      std::string(
```

Comment:
LGTM but do we need explicit conversion to std::string here?

Reply:
No. Not at all.